vocode/streaming/models/websocket.py

```diff
@@ -47,6 +47,7 @@ class AudioConfigStartMessage(
     input_audio_config: InputAudioConfig
     output_audio_config: OutputAudioConfig
     conversation_id: Optional[str] = None
+    session_id: Optional[str] = None
```

Comment:
probably missing something, but can we just use `conversation_id` for session management?